docs/taskruns.md

```diff
@@ -83,7 +83,18 @@ spec:
     name: read-task
 ```
 
-Or you can embed the spec of the `Task` directly in the `TaskRun`:
+You can also reference a `Task` contained in an OCI-compliant `Image`.
+
+```yaml
+spec:
+  taskRef:
+    name: read-task
+    image: docker.com/my-repo/my-image
```

Comment:
@bobcatfish as an image could (and would in the current experiments) be able to store more than one resource (aka several tasks, pipelines, …), `name` is required **as** it would be the way to get the right resource from the image. 

That said, we could also go the way of having only one resource per image — but I do like the idea to store multiples tasks in a single oci artifact :angel: